jazz_asset-event-handler/event.json

```diff
@@ -0,0 +1,20 @@
+{
+    "Records": [
+        {
+            "kinesis": {
+                "kinesisSchemaVersion": "1.0",
+                "partitionKey": "CREATE_ASSET",
+                "sequenceNumber": "49582744145785831874147508989126673490865670161183014914",
+                "data": "",
+                "approximateArrivalTimestamp": 1521632408.682
+            },
+            "eventSource": "aws:kinesis",
+            "eventVersion": "1.0",
+            "eventID": "shardId-000000000000:49582744145785831874147508989126673490865670161183014914",
+            "eventName": "aws:kinesis:record",
+            "invokeIdentityArn": "",
+            "awsRegion": "us-east-1",
```

Comment:
are we suppose to have these values

Reply:
Yes, event should be having a sample event payload.  I think we should follow this practice across board.  We usually keep even.json with defaults (empty)

Reply:
Storing sample event payload is a good idea. We should move this to test folder though.

Reply:
event.json is being created at the root level by default (when created the first template out of serverless.  Not sure this is a convention being followed or specific to serverless.  I think we should keep the serverless structure as it is.

Reply:
I don't think any of the comments in this thread are talking about deleting event.json. The payload in event.json is needs to be moved into test folder as it isn't related/required to the execution of this lambda.